readme.md

```diff
@@ -35,6 +35,13 @@ Returns a `Promise` that settles when localhost is ready.
 Type: `number`<br>
 Default: `80`
 
+#### options
+
+##### method
```

Comment:
Or should we use `httpMethod`?

Reply:
If we go for the method-named option, then yes, I think `httpMethod` is clearer.